src/public/stylesheets/theme-light.css

```diff
@@ -68,4 +68,11 @@ html {
     --link-color: blue;
 
     --mermaid-theme: default;
+
+    --banner-background-color-error: #DA321B;
+    --banner-background-color-info: #1bb7da;
+    --banner-background-color-warning: #daad1b;
+    --banner-background-color-success: #67da1b;
+    --banner-background-color-plain: var(--more-accented-background-color);
+    --banner-color: var(--main-text-color);
```

Comment:
I don't think it's needed to have this themable since it's for exceptional cases only and user shouldn't see it often.